lib/Mojolicious/resources/templates/mojo/debug.html.ep

```diff
@@ -100,18 +98,20 @@
     </header>
     <script>
       function mojoDrawer(handle, drawer) {
-        $(handle).on('click', function () {
-          $(drawer).slideToggle('slow');
-          const text = $(handle + ' div.tap').text();
+        const handleEl = document.querySelector(handle);
+        handleEl.addEventListener('click', () => {
+          drawerEl = document.querySelector(drawer);
```

Comment:
Where does `drawerEl` come from? I'm not seeing its scope.

Reply:
It was an error, I added 'use strict'; to the javascript to prevent this kind of error.